src/SFA.DAS.EmployerAccounts/Services/HttpService.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
```

Comment:
Again, could this be put in a shared package?

Reply:
Think we should review this as part of refactor work?

Reply:
AML-3616 covers this

Reply:
There's also a case to merge this and the RestService but again maybe refactor work?